.env.example

```diff
@@ -11,3 +11,5 @@ SOCIALHOME_DOMAIN=127.0.0.1:8000
 SOCIALHOME_HTTPS=False
 
 MOCHA_TESTS=True
+
+ADDITIONAL_APS=
```

Comment:
No need to have this in the example file since it defaults to none (and setting it is not a common use case).